src/riverroute/mosart_timemanager.F90

```diff
@@ -474,58 +478,30 @@ subroutine timemgr_restart(ncid, flag)
          end if
       end if
 
```

Comment:
Add comments about this no longer being something that is an either or with timemgr_init. Now, timemgr_init is always called. This also means timemgr_set should only be set with timemgr_init and here it's checked that it''s called.